packages/stable/src/api/functions/types.gen.ts

```diff
@@ -0,0 +1,445 @@
+   * @example {"MySecret":"***"}
+   */
+  secrets?: Record<string, string>;
+  status: FunctionStatus;
+}
+/**
+ * Cognite Function API error.
+ */
+export interface FunctionBuildError {
+  /**
+   * HTTP status code.
+   * @format int32
+   * @example 400
+   */
+  code: number;
+  /**
+   * Error message.
+   * @example Could not build function.
+   */
+  message: string;
+}
+export interface FunctionCall {
+  /** The number of milliseconds since 00:00:00 Thursday, 1 January 1970, Coordinated Universal Time (UTC), minus leap seconds. */
+  endTime?: EpochTimestamp;
+  /** Error occuring due to user's function code. */
+  error?: FunctionCallError;
+  /** A server-generated ID for the object. */
+  functionId: CogniteInternalId;
+  /** A server-generated ID for the object. */
+  id: CogniteInternalId;
+  /** The number of milliseconds since 00:00:00 Thursday, 1 January 1970, Coordinated Universal Time (UTC), minus leap seconds. */
+  scheduledTime?: EpochTimestamp;
+  /** A server-generated ID for the object. */
+  scheduleId?: CogniteInternalId;
+  /** The number of milliseconds since 00:00:00 Thursday, 1 January 1970, Coordinated Universal Time (UTC), minus leap seconds. */
+  startTime: EpochTimestamp;
+  /** Status of the function call. */
+  status: FunctionCallStatus;
+}
+export type FunctionCalledResponse = FunctionCall;
+/**
+ * Error occuring due to user's function code.
+ */
+export interface FunctionCallError {
+  /**
+   * Error message.
+   * @example Could not authenticate.
+   */
+  message: string;
+  /**
+   * Stack trace of exception, useful for debugging.
+   * @example Cannot assign foo to bar.
+   */
+  trace?: string;
+}
+export interface FunctionCallFilter {
+  /** FunctionCallFilter */
+  filter?: {
+    scheduleId?: CogniteInternalId;
+    status?: FunctionCallStatus;
+    startTime?: EpochTimestampRange;
+    endTime?: EpochTimestampRange;
+  };
+}
+export type FunctionCallListScope = FunctionCallFilter & LimitList & Cursor;
+export type FunctionCallListWithCursorResponse = FunctionCallsWithCursor;
+export interface FunctionCallLogEntry {
+  /**
+   * Single line from stdout / stderr.
+   * @example Did something great
+   */
+  message?: string;
+  /** The number of milliseconds since 00:00:00 Thursday, 1 January 1970, Coordinated Universal Time (UTC), minus leap seconds. */
+  timestamp?: EpochTimestamp;
+}
+export interface FunctionCallLogResponse {
+  items: FunctionCallLogEntry[];
+}
+export interface FunctionCallRequest {
+  // biome-ignore lint/suspicious/noExplicitAny: couldn't find correct type definition.
+  data?: any;
+  // biome-ignore lint/suspicious/noExplicitAny: couldn't find correct type definition.
```

Comment:
Are these comments auto-generated 🤔 ?

Reply:
E.g. `nonce` here should be a string

Reply:
Heh, no, will update the types, thanks :)